src/repos/ArticleRepo.ts

```diff
@@ -5,24 +5,34 @@ import { DEFAULT_LIMIT, MAX_NUM_DAYS_OF_TRENDING_ARTICLES } from '../common/cons
 import { PublicationModel } from '../entities/Publication';
 
 const getArticleByID = async (id: string): Promise<Article> => {
-  return ArticleModel.findById(new ObjectId(id));
+  const article = ArticleModel.findById(new ObjectId(id));
+  if((await article).filtered){
+    return article;
+  }
+  return null;
```

Comment:
doesn't this return `null` if `filtered=false`?

Reply:
good catch lol